diagnostic.md

```diff
@@ -100,11 +135,23 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH 'http://www.example.com/donuts/10' \
+  --header "Content-Type: application/json" \
+  --data $'{
+    "donut": {
+      "name": $NAME,
+      "price": $PRICE
+    }
+  }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function (event) {
+  event.preventDefault();
+  libraryApi.update(event.target)
```

Comment:
1/2. I see what you wanted to do here but this is not an ajax request, the libraryApu.update() function that this calls would be the ajax request